modules/virtualization/microvm/guivm.nix

```diff
@@ -0,0 +1,128 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  configHost = config;
+  waypipe-ssh = pkgs.callPackage ../../../user-apps/waypipe-ssh {};
+  guivmBaseConfiguration = {
+    imports = [
+      ({
+        lib,
+        pkgs,
+        ...
+      }: {
+        ghaf = {
+          users.accounts.enable = lib.mkDefault configHost.ghaf.users.accounts.enable;
```

Comment:
Referring to this comment - https://github.com/tiiuae/ghaf/pull/243/files#r1308262741
GUIVM should share the user account with the AppVMs but should NOT share the user account with host.
I would prefer to have this user account separation either:
- part of this refactoring PR or
- ticket to track that it'll get implemented

Reply:
SP-2869 is tracking this